### PR TITLE
Adds bolt and tests from serial-ghost-tables PR

### PR DESCRIPTION
**Pull request by gfiorav**
Not updating `NEWS.md` because it's an unused tool atm.

Please CR @juanignaciosl

### CODE REVIEW

File: lib/carto/bolt.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# encoding utf-8
+
+module Carto
+  class Bolt
+    DEFAULT_REDIS_OBJECT = $users_metadata
+    DEFAULT_TTL_MS = 10000
```
Comment (juanignaciosl) [inline lib/carto/bolt.rb]: Wasn't 10s a little too low?

Comment (gfiorav) [inline lib/carto/bolt.rb]: It's a default!

What would be a better choice? 60 secs?

**Comment by juanignaciosl** [comment]
:ok_hand:

File: lib/carto/bolt.rb (hunk 1 above)
Comment (juanignaciosl) [inline lib/carto/bolt.rb]: Ok, let it as a default if you're doing some measures for ghost tables later. I've tried to work out a conclusion based on ghost tables logging but I haven't been able to get any trustworthy value, so up to you.

**Pull request merged**